build-logic/src/main/kotlin/kotlin-inject.multiplatform.gradle.kts

```diff
@@ -34,8 +34,9 @@ kotlin {
                 dependsOn(nativeTest)
             }
         }
+
         // Ensure xml test reports are generated
-        val jvmTest by tasks.getting(Test::class) {
+        tasks.named("jvmTest", Test::class).configure {
```

Comment:
This case is also not just about the unused variable warning, but because `getting` doesn't use task configuration avoidance. I'll leave my changes for this, unless you feel I should revert it.

Reply:
Oh dang I thought it did since it was a 'newer' api.